projects/arlas-components/src/lib/pipes/format-legend/format-legend.pipe.ts

```diff
@@ -80,33 +81,37 @@ export class FormatLegendPipe implements PipeTransform {
       const field = this.getField(valueSplit);
       const normalised = parts[1] ?? '';
       params = this.buildInterpolatedParams(params, metric, field, normalised);
+    } else if (parts[0].endsWith('_arlas__color')){
+      params.translateKey = params.translateKey.replace('_arlas__color', '');
+    } else if(params.translateKey === 'Heatmap-density') {
```

Comment:
Why not directly translate Heatmap-density ?

Reply:
if for some reason we want keep heatmap-density its better to use the good translation key than use heatmap ?

Reply:
As you prefer, it seems like an unnecessary extra step to me